### PR TITLE
update svdb merge

### PR DESCRIPTION
**Pull request by ramprasadn**
## PR checklist

This PR updates svdb/merge module to work with '--notag' option

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
    - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd`

### CODE REVIEW

File: modules/svdb/merge/main.nf [hunk 1]
```diff
@@ -22,8 +22,14 @@ process SVDB_MERGE {
     def args   = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
     def input  = ""
-    for (int index = 0; index < vcfs.size(); index++) {
-        input += " ${vcfs[index]}:${priority[index]}"
+    if(priority) {
+        prio = "--priority ${priority.join(',')}"
```
Comment (projectoriented) [inline modules/svdb/merge/main.nf:26]: Nice, I think with this you can remove `--priority ${priority.join(',')} \\ `in the script block right?

Comment (ramprasadn) [reply modules/svdb/merge/main.nf:26]: Yupe! Good catch!

**Comment by projectoriented** [comment]
Hmm, I'm noticing `--priority` https://github.com/J35P312/SVDB#merge is an optional arg & I understand why you put that there but let's wait for @jemten to comment

**Comment by ramprasadn** [comment]
I am not sure I understand your comment @projectoriented. If priority isn't set i.e. if its an empty list, then it should be fine, no?

**Comment by projectoriented** [comment]
Yup! I mean in terms of L12

File: modules/svdb/merge/main.nf [hunk 2]
```diff
@@ -22,14 +22,20 @@ process SVDB_MERGE {
...
+        for (int index = 0; index < vcfs.size(); index++) {
+            input += " ${vcfs[index]}:${priority[index]}"
+        }
+    } else {
+        prio = ""
+        input = "${vcfs.join(" ")}"
```
Comment (jemten) [inline modules/svdb/merge/main.nf:32]: You could make this the default option and then just overwrite these variabels if `priority` is set. That is if you want to save a few lines of code. Doesn't really matter 😄

**Review by jemten — APPROVED** [review]
Looks good to me. Wrote a small comment but that is just a style thing. Was there something specific I should look at @projectoriented, @ramprasadn?

File: modules/svdb/merge/main.nf (hunk 2 above)
Comment (ramprasadn) [reply modules/svdb/merge/main.nf]: Good point! Changed it now 😄

**Pull request merged**